public/src/main/java/com/revenuecat/purchases/parceler/JSONObjectParceler.kt

```diff
@@ -5,17 +5,13 @@ import kotlinx.android.parcel.Parceler
 import org.json.JSONObject
 
 /** @suppress */
-internal object JSONObjectParceler :
-    Parceler<JSONObject> {
+internal object JSONObjectParceler : Parceler<JSONObject> {
 
     override fun create(parcel: Parcel): JSONObject {
         return JSONObject(parcel.readString())
     }
 
     override fun JSONObject.write(parcel: Parcel, flags: Int) {
-        val field = JSONObject::class.java.getDeclaredField("jsonObject")
-        field.isAccessible = true
-        val value = field.get(this).toString()
-        parcel.writeString(value)
+        parcel.writeString(this.toString())
```

Comment:
I don't think we have to throw a hotfix just for this. It's been like this forever and only affects the `jsonObject` inside the `PurchaserInfo` when passing a `PurchaserInfo` between activities.

I will make another PR since it's unrelated to Amazon